pkg/local_object_storage/blobstor/put.go

```diff
@@ -42,6 +44,10 @@ func (b *BlobStor) Put(prm common.PutPrm) (common.PutRes, error) {
 			res, err := b.storage[i].Storage.Put(prm)
 			if err == nil {
 				logOp(b.log, putOp, prm.Address, b.storage[i].Storage.Type(), res.StorageID)
+			} else if errors.Is(err, common.ErrNoSpace) {
```

Comment:
Do we want to get `ErrNoPlaceFound` if all of them fail this way or `ErrNoSpace`?

Reply:
ofc `ErrNoSpace` is clearer, i was confused by `ErrNoPlaceFound` name (look like no space)

done